packages/http/src/validator/deserializers/index.ts

```diff
@@ -11,14 +11,15 @@ import type { IHttpNameValue, IHttpNameValues } from '../../';
 export const header: Dictionary<deserializeFn<IHttpNameValue>, 'simple'> = { simple };
 export const query: Dictionary<
   deserializeFn<IHttpNameValues>,
-  'form' | 'spaceDelimited' | 'pipeDelimited' | 'commaDelimited' | 'deepObject' | 'simple'
+  'form' | 'spaceDelimited' | 'pipeDelimited' | 'commaDelimited' | 'deepObject' | 'simple' | 'unspecified'
 > = {
   form,
   spaceDelimited: delimited('%20'),
   pipeDelimited: delimited('|'),
   commaDelimited: delimited(','),
   deepObject,
   simple: form,
+  unspecified: form,
```

Comment:
since form is the default value for query params, this seemed like the best option, but open to opinions!